src/navigation/ActivationStack.tsx

```diff
@@ -104,7 +119,7 @@ const ActivationStack: FunctionComponent = () => {
 
   return (
     <Stack.Navigator screenOptions={screenOptions}>
-      {activationSteps.map((step, idx) => {
+      {activationSteps?.map((step, idx) => {
```

Comment:
Whats the need for the optional operator here? wouldnt activationSteps always be defined as an array?

Reply:
For some reason Typescript thinks it could be undefined. Is there a workaround for that?

Reply:
I would look into the type signature of Platform.select and see if theres a thread to follow there.

Reply:
Good call, it needed a default.